🛠️ Repo: object-shorthand lint rule failure on main

After a fresh clone and `pnpm i; pnpm build` of the repo:

```plaintext
 ~/repos/eslint-plugin-jest $ yarn lint

/Users/josh/repos/eslint-plugin-jest/src/rules/prefer-jest-mocked.ts
  50:9  warning  Expected property shorthand  object-shorthand

✖ 1 problem (0 errors, 1 warning)
  0 errors and 1 warning potentially fixable with the `--fix` option.
```

Looks like it's not failing any builds because it's a warning.

--- a/src/rules/prefer-jest-mocked.ts
+++ b/src/rules/prefer-jest-mocked.ts
@@ -47,7 +47,7 @@ export default createRule({
       );
 
       context.report({
-        node: node,
+        node,
         messageId: 'useJestMocked',
         fix(fixer) {
           return fixer.replaceText(node, `jest.mocked(${fnName})`);

